CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Requests with empty body and application/json Content-Type Header will now
   return 400 error instead of 500 error.
   [cyberark/conjur#1968](https://github.com/cyberark/conjur/issues/1968)
+- Conjur now properly sets the 'operation' field of a secret update audit message.
```

Comment:
This needs a link to the issue for this change, and it could speak more to the user impact (if I am a user, why do I care about this change?)

Reply:
Updated message and still waiting for the issue number, will update when we will have the issue.